.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+        target-framework: [net48, netcoreapp3.1]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python environment
+      uses: actions/setup-python@v1.1.1
+    - name: Restore Python packages
+      run: |
+        python -m pip install -U pip
+        python -m pip install -r ${{ github.workspace }}\ManualGenerator\requirements.txt
+    - name: Setup NuGet.exe for use with actions
+      uses: NuGet/setup-nuget@v1.0.2
+    - name: Restore NuGet packages
+      run: nuget restore ${{ github.workspace }}\ThScoreFileConverter.sln
+    - name: Setup MSBuild.exe
+      uses: warrenbuckley/Setup-MSBuild@v1
+    - name: Build
+      run: msbuild ${{ github.workspace }}\ThScoreFileConverter.sln /m /verbosity:minimal /p:Configuration=${{ matrix.configuration }} /p:TargetFrameworks=${{ matrix.target-framework }}
+    - name: Generate coverage report
+      run: |
+        & ${{ github.workspace }}\OpenCover.ps1 ${{ github.workspace }}\ThScoreFileConverterTests\bin\${{ matrix.configuration }}\${{ matrix.target-framework }}\coverage.xml
+      shell: powershell
+      if: ${{ matrix.configuration == 'Debug' }}
```

Comment:
Syntax error:
```
- Your workflow file was invalid: The pipeline is not valid. .github/workflows/main.yml (Line: 35, Col: 11): Unrecognized named-value: 'matrix'. Located at position 1 within expression: matrix.configuration == 'Debug',.github/workflows/main.yml (Line: 41, Col: 11): Unrecognized named-value: 'matrix'. Located at position 1 within expression: matrix.configuration == 'Debug'
```
See: https://help.github.com/en/actions/reference/contexts-and-expression-syntax-for-github-actions#about-context-and-expressions